website/content/docs/secrets/databases/redis.mdx

```diff
@@ -0,0 +1,122 @@
+---
+layout: docs
+page_title: Redis - Database - Secrets Engines
+description: |-
+  Redis is one of the supported plugins for the database secrets engine.
+  This plugin generates database credentials dynamically based on configured
+  roles for the Redis database.
+---
+
+# Redis Database Secrets Engine
+
+Redis is one of the supported plugins for the database secrets engine. This
+plugin generates database credentials dynamically based on configured roles for
+the Redis database.
+
+See the [database secrets engine](/docs/secrets/databases) docs for
+more information about setting up the database secrets engine.
+
+## Capabilities
+
+| Plugin Name                 | Root Credential Rotation | Dynamic Roles | Static Roles | Username Customization |
+| --------------------------- | ------------------------ | ------------- | ------------ | ---------------------- |
+| `redis-database-plugin`     | Yes                      | Yes           | Yes          | No                     |
+
+## Setup
+
+1.  Enable the database secrets engine if it is not already enabled:
+
+```bash
```

Comment:
Two small comments:

The numbering for the setup list can actually be all `1.` instead of 1. 2. 3. etc. It renders the sequentially numbers automatically. What's nice about using this is that in the future if more steps are added, the author doesn't have to go back and reorder a bunch of numbers. The list under Dynamic roles and Static roles look to already be using this, if you want to look at how it renders as an example.

The language tag to use for shell code blocks is:
~~~
```shell-session
   ...
```
~~~
according to the README in the website directory of Vault. (Not sure the reason why if I'm honest with you, but there probably is one)